src/pages/project/project_wizard/ProjectForm.tsx

```diff
@@ -132,6 +145,7 @@ export const ProjectForm = ({ project, suggestedProjectManager }: ProjectFormPro
                   onClickNext={() => onClickArrow(tabNumber + 1)}
                   onClickPrevious={() => onClickArrow(tabNumber - 1)}
                   onClickLast={() => onClickArrow(ProjectTabs.Connections)}
+                  openedInModal={toggleModal !== undefined}
```

Comment:
for å matche sjekkene på linje 68 og 88
```suggestion
                  openedInModal={toggleModal}
```